### PR TITLE
Set angular to bring dimensions on explorer page

### PR DESCRIPTION
**Pull request by nanchenchen**
for #18

### CODE REVIEW

File: msgvis/static/SparkQs/controllers.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+    ]);
+
+    var requires = ['$scope'
+      ];
+    var DimensionController = function ($scope) {
+
+        $scope.$watch(function(){
+            console.log("digest called");
+        });
```
Comment (michaelbrooks) [inline msgvis/static/SparkQs/controllers.js]: Is this for anything in particular?

File: msgvis/static/SparkQs/controllers.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        $scope.dimension_groups = [
+            {
+                "group_name": "Time",
+                "dimensions": [
+                    {
+                        "name": "Time",
+                        "type": "TimeDimension"
+                    },
+                    {
+                        "name": "Timezone",
```
Comment (michaelbrooks) [inline msgvis/static/SparkQs/controllers.js]: Can we use "key" instead of "name" for consistency?

Comment (michaelbrooks) [inline msgvis/static/SparkQs/controllers.js]: Actually lets use something like the [dimension API object](http://textvisdrg.readthedocs.org/en/latest/apps/api.html#msgvis.apps.api.serializers.DimensionSerializer) if that makes sense.

**Comment by nanchenchen** [comment]
This should be good to go but instead of having it merges to master, I want to make a branch for front-end stuffs so that I do not need to wait for merge to master. Will close this for now.

**Pull request closed without merging**